modules/dlscript/dl_script.cpp

```diff
@@ -320,10 +390,10 @@ void DLScript::set_library(Ref<DLLibrary> p_library) {
 		return;
 #endif
 	if (library.is_valid()) {
-		Error initalize_status = library->_initialize_handle(!ScriptServer::is_scripting_enabled());
+		Error initalize_status = library->_initialize();
 		ERR_FAIL_COND(initalize_status != OK);
 		if (script_name) {
-			script_data = library->get_script_data(script_name);
+			script_data = library->library->scripts[script_name];
```

Comment:
I can give it a pointer to itself named `library`, that way you can chain as many of those as you want!